.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -8,8 +8,15 @@ variables:
 pool:
   name: $(POOL_NAME)
 
-
 parameters:
+  - name: useOverrides
+    displayName: Use component overrides
+    type: boolean
+    default: false
+  - name: overrideBranch
+    displayName: Branch in aks-rp to use for overrides
+    type: string
+    default: master
```

Comment:
this is fine for default, though I think it's worth noting that in these situations I don't think we'll be overriding from master, since otherwise we'd have to wait for the overrides to be merged into master before running the corresponding build

Reply:
Will add that to the docs when I write then :)